docs/content/docs/dev/table/jdbcDriver.md

```diff
@@ -51,16 +55,79 @@ You can also add dependency of Flink JDBC driver in your maven or gradle project
     </dependency>
 ```
 
-## Use with a JDBC Tool
-### Use with Beeline
+## JDBC Clients
+
+The Flink JDBC driver is not included with the Flink distribution. You can download it from [Maven](https://repo.maven.apache.org/maven2/org/apache/flink/flink-sql-jdbc-driver-bundle/). 
+
+You may also need the [SLF4J](https://repo1.maven.org/maven2/org/slf4j/slf4j-api/) (`slf4j-api-{slf4j.version}.jar`) jar.
+
+### SQLLine
+
+[SQLLine](https://github.com/julianhyde/sqlline) is a lightweight JDBC command line tool that supports general JDBC drivers. 
+
+To use SQLLine you will need to clone [the GitHub repository](https://github.com/julianhyde/sqlline) and compile the project first (`./mvnw package -DskipTests`).
+
+1. Download the following JARs and add them both to the `target` directory of SQLLine project:
+   1. [Flink JDBC Driver](https://repo.maven.apache.org/maven2/org/apache/flink/flink-sql-jdbc-driver-bundle/) (`flink-jdbc-driver-bundle-{VERSION}.jar`) 
+   2. [SLF4J](https://repo1.maven.org/maven2/org/slf4j/slf4j-api/) (`slf4j-api-{slf4j.version}.jar`)
+2. Run SQLLine with command `./bin/sqlline` 
+3. From SQLLine, connect to a Flink SQL gateway using the `!connect` command. 
+
+    Since the Flink SQL gateway currently ignores user names and passwords just leave them empty.
+
+    ```sql
+	sqlline version 1.13.0
```

Comment:
where did you get it?
The latest sqlline version is 1.12.0 (i was a release manager there btw =))

Reply:
I built it from the github repo per the doc instructions :) https://github.com/apache/flink/pull/23743/files/940465d9294f4cbc4e64b06610bfe7d47a6a1ee7#diff-1f75f7a315c8363270172df0bf6850eac1f9ce6c3a4b9f8ee8c8d90ce75bda68R68

Reply:
based on https://github.com/julianhyde/sqlline/blob/f7889ec6058ce404489ed0443755f5885e6258eb/pom.xml#L13
it's pretty likely it's impossible to get 1.13.0 without extra changes
it's possible to get only `1.13.0-SNAPSHOT` (or did I miss anything?)
anyway since it is not officially released we can not refer to it, only to something which is announced and released officially

Reply:
ah yes, you're completely right, sorry. It is `1.13.0-SNAPSHOT`. I'll update the PR.